File: frontpage/custom_script/word2html.py
'''
Created  on 2015-11-27
Modify  on  2015-11-30
@author:lianshifeng
'''
import win32com 
import os 
import sys
import string 
from win32com.client import Dispatch,constants

reload(sys)
sys.setdefaultencoding('utf-8')


def word2html(filename):
    word = win32com.client.DispatchEx('Word.Application')
    word.Visible = 0   
    doc = word.Documents.Open(filename)
    filename = filename.split(".")
    if (filename[-1] == "doc" or filename[-1] == "docx"):
        filename[-1]="html"
        htmlname=string.join(filename,".")
    doc.SaveAs(htmlname,10)
    doc.Close() 
    word.Quit()  

if __name__ =='__main__':
    filename = sys.argv[1]
    if filename: 
        word2html(filename)

File: frontpage/custom_script/word2html.php
 <?php 


            $wfilepath='D:/wamp/www/New-Bee/uploads/1.doc';
            $word = new com("word.application",null,CP_UTF8) or die("wu fa dakai zhege wenjianjia"); 
            $word->visible = 1 ;    
            $word->Documents->Open($wfilepath)or die("wu fa dakai zhege wenjianjia");  
            $htmlpath=substr($wfilepath,0,-4);   
            $word->ActiveDocument->SaveAs($htmlpath,8);   
            $word->quit(0); 
   ?>